modules/local/sentieon/dnamodelapply.nf

```diff
@@ -0,0 +1,49 @@
+process SENTIEON_DNAMODELAPPLY {
+    tag "$meta.id"
+    label 'process_high'
+    label 'sentieon'
+
+    input:
+    tuple val(meta), path(vcf), path(vcf_idx)
+    path fasta
+    path fai
+    path ml_model
+
+    output:
+    tuple val(meta), path("*_dnascope_ml.vcf")     , emit: vcf
+    tuple val(meta), path("*_dnascope_ml.vcf.idx") , emit: vcf_index
+    path "versions.yml"                            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    sentieon driver \\
+        -t $task.cpus \\
+        -r $fasta \\
+        --algo DNAModelApply \\
+        --model $ml_model \\
+        -v $vcf \\
+        ${prefix}_dnascope_ml.vcf
```

Comment:
Is it possible to have this sentieon produce bgzipped output?

Reply:
It is possible; sentieon uses a separate module (util vcfconvert) for compression. However, that module doesn't accept input from terminal so using pipe is not an option, and so it has to be a separate step in the pipeline. 

Also, I didn't focus so much on the compression here because vcf files generated here will have to be combined in the next step to generate a single multisample vcf in case of duos and trios. Since we will only publish that multisample vcf, it made sense to compress that file directly and not these individual sample vcfs. What do you think?

Reply:
Just adding .gz to file extension did the trick! 😄